examples/playbooks/example.yml

```diff
@@ -10,7 +10,7 @@
       action: ansible.builtin.command echo {{this_variable}} is not set in this playbook
```

Comment:
I am sure you did not want to touch this file!

Reply:
When I opened this to check made a few changes and reverted them but then on save it started removing some spaces and when I reverted this change strangely pre-commit started picking this change and kept on removing the spaces